### DISCUSSION TITLE
Mos.app can't be opened because Apple cannot check it for malicious software

### DISCUSSION THREAD

**Discussion by HillelTheElder** [Caldis/Mos#463] [Q&A]
<img width="262" alt="Screen Shot 2022-01-12 at 17 32 09" src="https://user-images.githubusercontent.com/34669589/149239645-4f205d27-91c0-40a4-b083-e347175fdef4.png">

Mos.app can't be opened because Apple cannot check it for malicious software. This software needs to be updated. Contact the developer for more information.

Plz halp?

**Comment by widcha** [comment]
you can try by going to the application folder and find the MOS app, right-click and then click open.
it works for me.

**Reply by Abzoozy** [reply]
@widcha Thank you! this has worked for me until this error gets fixed.

**Reply by arnabdotpy** [reply]
Thanks, that worked for me as well.

**Comment by Rydvansky** [comment]
Why this issue occurs?

**Comment by diversityX** [comment]
Still an issue, and a right click doesn't solve it in Monterey

**Comment by Valamorde** [comment]
Maybe late to the party, but once you get this message you can go to System Settings > Privacy & Security and scroll down; you should see a prompt to allow the application to run.

**Comment by mikequang** [comment]
Setting > Privacy & Security > Accessibility, then allow Mos

**Comment by ar-dtag** [comment]
Both options are not working for me. Any idea how to open Mos in MacOS 15.2?

**Reply by thomas141** [reply]
go to system setting security and scroll down to the bottom. you will see a force open option

**Comment by emilioferreyra** [comment]
Open System Settings > Privacy & Security and scroll down. Then you'll see this message:

![SCR-20250302-qfcv](https://github.com/user-attachments/assets/494238d0-f7b5-481f-b0e6-8b8ed1107ed6)

And a confirmation pop up window will open, so you can chouse open anyway again:

![SCR-20250302-qiaz](https://github.com/user-attachments/assets/cc5ca119-88b9-4b5d-abc2-33aa3e23aa5f)

**Reply by ar-dtag** [reply]
On my system that was not working. The open Anyway option is not displayed. But to remove MOS quarantine status helped.
See https://github.com/Caldis/Mos/wiki/If-the-App-not-work-properly.

Thx!
